jdbc/src/main/java/camp/nextstep/jdbc/core/SqlExecuteFunction.java

```diff
@@ -0,0 +1,7 @@
+package camp.nextstep.jdbc.core;
+
+import java.sql.SQLException;
+
+public interface SqlExecuteFunction<T> {
```

Comment:
```suggestion
@FunctionalInterface
public interface SqlExecuteFunction<T> {
```

요놈도요!

run 과 apply 라는 메서드명을 보니 코틀린 생각이 나는군요 ㅋㅋ